client/src/components/AboutDialog.vue

```diff
@@ -70,8 +70,7 @@
 import Vue from 'vue';
 
 import AExt from '@/components/AExt.vue';
-
-const GIT_SHA: string = process.env.VUE_APP_GIT_SHA || '';
+import { gitSha as GIT_SHA } from '@/environment';
```

Comment:
No need to shift it to `GIT_SHA` (for consistency with the other uses).

Reply:
This is done to follow linting guidelines (and I only noticed it from the linter), as without this renaming, the `gitSha` computed prop shadows the original `gitSha` import.

Reply:
Ah I see. This seems to be implemented poorly (my fault). I will open an issue to fix this (by passing in the SHA value as a prop).